zerver/lib/bugdown/__init__.py

```diff
@@ -146,6 +146,7 @@ def add_a(root, url, link, title=None, desc=None,
         a.set("data-id", data_id)
     img = markdown.util.etree.SubElement(a, "img")
     img.set("src", url)
+    img.set("alt", title)
```

Comment:
The WebAIM site has some [guidelines](http://webaim.org/articles/gonewild/#title) on this.  Basically, the title attribute should only be used for additional information that isn't required for a11y purposes (and will often be skipped by screen readers).